PYMYSQL_DUPLICATE_ERROR = 1062
